apps/test/netsim/NetSimTable.js

```diff
@@ -0,0 +1,231 @@
+var testUtils = require('../util/testUtils');
+var assert = testUtils.assert;
+var NetSimTable = testUtils.requireWithGlobalsCheckBuildFolder('netsim/NetSimTable');
+
+var assertEqual = function (left, right) {
+  assert(left === right, '' + left + '===' + right);
```

Comment:
`left + ' === '` should already cast to string, I don't think you need the initial `'' +`